repository.tf.sample

```diff
@@ -24,6 +24,7 @@ resource "github_branch_protection" "example_master" {
 
   required_status_checks {
     strict = true
+    contexts = ["example-pr"]
```

Comment:
this should probably be `["concourse-ci/example-pr"]`

Reply:
Agreed.